snap/info.go

```diff
@@ -31,6 +31,19 @@ import (
 	"github.com/ubuntu-core/snappy/timeout"
 )
 
+// FIXME: make this a struct and add a special unmarshal that will deal
+//        with both int and string
```

Comment:
I wouldn't worry about making it a struct. We can make Revision as-is have an unmarshal method.